documentation/doc.html

```diff
@@ -585,8 +586,48 @@ <h4>The alert offers four severity levels that set a distinctive icon and color.
           </div>
          
           <hr/>
-
-
+          <h1 id="snackbar">SnackBar</h1>
+          <p>Snackbars provide brief notifications. The component is also known as a toast.</p>
+          <p>Click here 👇 to see live demo modal.</p>
+          <div class="snackbar">
+              <button class="base-line-btn normal-btn btn-login" id="base-line-btn">Base line</button>
+              <button class="leading-line-btn normal-btn btn-login" id="leading-line-btn">Leading</button>
+              <button class="stacked-line-btn normal-btn btn-login" id="stacked-line-btn">Stacked</button>
+          </div>
+          <div class="base-line-snackbar">
+            <section class="snackbar2">
```

Comment:
try using better class name(which is self-explanatory)

Reply:
Done.